server/index.js

```diff
@@ -0,0 +1,32 @@
+import {} from 'dotenv/config';
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import path from 'path';
+
+const app = express();
+const port = process.env.PORT || 1337;
+const assets = path.join(__dirname, '../client');
+
+app.listen(port, () => console.log(`express is listening on ${port}`)); // eslint-disable-line
+// disable for global requires since airbnb doesn't like them being in a block
+if (process.env.NODE_ENV === 'development') {
+  /* eslint-disable */
+  const config = require('../webpack.dev.js');
+  const compiler = require('webpack')(config);
+  app.use(require('webpack-dev-middleware')(compiler, {
```

Comment:
Pretty cool to have this for in-memory generation instead of writing to file. This is only used for development though, right?

For production, we would be generating something like the "bundle.js" file. Does this jive with your understanding as well?

Reply:
Yeah this is only for development. Makes it easy for us to make changes on the fly and see them reflected right away. Hot reload should just reload the component that's been changed which is pretty cool. For prod, I'll have another webpack config that and script that will do the webpack bundling as well as transpile our backend code into ES5. 

Right now, we're using babel-node with babel inline to transpile our code on the fly. Easy dev work but not so great in prod since it has a lot of overhead.